### DISCUSSION TITLE
/v2/md-enhance/zh/guide/image.html

### DISCUSSION THREAD

**Discussion by giscus** [vuepress-theme-hope/giscus-discussions#204] [Announcements]
# /v2/md-enhance/zh/guide/image.html

VuePress 的 Markdown 增强插件

https://vuepress-theme-hope.github.io/v2/md-enhance/zh/guide/image.html

**Comment by QiyuLuo** [comment]
您好，发现图片居中需要手动对每个图片设置center，请问能否通过配置，设置图片自动居中。

**Reply by Mister-Hope** [reply]
自己写个css样式。。。。。
